src/main/java/ortus/boxlang/runtime/util/ZipUtil.java

```diff
@@ -83,7 +83,11 @@ public enum COMPRESSION_FORMAT {
 	 * @param overwrite         Whether to overwrite the destination file if it already exists, default is false
 	 * @param prefix            String added as a prefix to the final archive. The string is the name of a subdirectory in which the entries are added to exclusively. Only works for zip archiving.
 	 * @param filter            A regex or BoxLang function or Java Predicate to apply as a filter to the extraction
+	 * @param recurse           Whether to recurse into subdirectories, default is true
+	 * @param compressionLevel  The compression level to use for the compression. Default is 6, which is a good balance between speed and compression ratio.
```

Comment:
This also says the default is not what we have it set to above

Reply:
Oops 😬